### PR TITLE
Added Firebase Crash analytics

### PR DESCRIPTION
**Pull request by codepoet2017390**
- Initialized Firebase in the app and connected to web console
- Added firebase crashlytics to the app
Login to firebase console for monitoring

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -34,6 +34,9 @@
             android:label="@string/app_name"
             android:theme="@style/AppTheme.NoActionBar">
         </activity>
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="637dc807ac8096c2df2f153c1fb89fa7a61ee415" />
```
Comment (isears) [inline app/src/main/AndroidManifest.xml:39]: Is this safe to expose on a public repo? Why do we need both a fabric api key and a firebase api key?

Comment (codepoet2017390) [reply app/src/main/AndroidManifest.xml:39]: https://github.com/zooniverse/vox/issues/32

Comment (codepoet2017390) [reply app/src/main/AndroidManifest.xml:39]: Also Firebase is not just about Crash analytics, it comes with many other features like general usage patterns, auth and much more. Now firebase itself uses fabric for providing crash analytics and makes you register separately for crash analytics with fabric.
Though after some time both will be merged together but still API key will be required for both.

Comment (isears) [reply app/src/main/AndroidManifest.xml:39]: I saw the zooniverse link, but that seemed to refer to the firebase api key, not the fabric api key. Most sources I could find regarding the fabric api key are implying that it's not safe to release, but I know the recent project merger may be changing things.

Comment (codepoet2017390) [reply app/src/main/AndroidManifest.xml:39]: We can go for something like this 
https://github.com/codepath/android_guides/wiki/Storing-Secret-Keys-in-Android

**Comment by isears** [comment]
superseded by #16

**Pull request closed without merging**